eaw-types-emmyluadoc.lua

```diff
@@ -279,6 +315,22 @@ end
 function GameScoringType.Get_Build_Cost()
 end
 
+---@class CommandBlock
+local CommandBlock = {}
```

Comment:
IIRC the table definition also has to be included in the return list at the end of the file, unless EmmyLua has been updated to pick up on local class declarations without an explicit return.
Might also be a problem with the LSP-based implementaiton for Sublime Text but including it shouldn'T hurt unless someone actually `require`s the file.